package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "dilophos",
+    "version": "0.1.0",
+    "description": "Dilophosaurus Operating System (DilophOS)",
```

Comment:
Are we not adding "keywords", "repository", and "homepage" to this package.json?

Reply:
Ah, good catch